Holovibes/includes/data_struct/batch_input_queue.hh

```diff
@@ -74,7 +82,7 @@ class BatchInputQueue final : public DisplayQueue
      * Called by the consumer.
      * Call copy multiple with nb_elts == batch_size_
      *
-     * \param dest The destination queue
+     * \param[in] dest The destination queue
```

Comment:
`cuda_kind` param is missing and dest should be param[out] if it is used as output